src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -547,6 +547,39 @@ GUI_TEST_CLASS_DEFINITION(test_7152) {
     CHECK_SET_ERR(bottomRight == "11/40/35", "Bottom right position is wrong: " + bottomRight);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7154) {
+    // 1. Open "_common_data/genbank/Smc3_LOCUS_19_45436_bp_DNA_HTG_4_changed.gbk".
+    GTFileDialog::openFile(os, testDir + "_common_data/genbank/Smc3_LOCUS_19_45436_bp_DNA_HTG_4_changed.gbk");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+
+    // 2. Create annotation #1
+    GTUtilsDialog::waitForDialog(os, new CreateAnnotationWidgetFiller(os, false, "grpA", "annA", "complement(10.. 20)"));
+    GTKeyboardDriver::keyClick('n', Qt::ControlModifier);
+
+    // 3. Create annotations #2
+    GTUtilsDialog::waitForDialog(os, new CreateAnnotationWidgetFiller(os, false, "grpB", "annB", "complement(30.. 40)"));
+    GTKeyboardDriver::keyClick('n', Qt::ControlModifier);
+
+    // 7. Drag&drop annotation #1 to group #2
+    QTreeWidgetItem* annA = GTUtilsAnnotationsTreeView::findItem(os, "annA");
+    QTreeWidgetItem* annB = GTUtilsAnnotationsTreeView::findItem(os, "annB");
+    QTreeWidgetItem* grpA = annA->parent();
+    QTreeWidgetItem* grpB = annB->parent();
+    QPoint pointA = GTUtilsAnnotationsTreeView::getItemCenter(os, "annA");
+    QPoint pointGrpA = GTTreeWidget::getItemCenter(os, grpA);
+    QPoint pointGrpB = GTTreeWidget::getItemCenter(os, grpB);
+    GTThread::waitForMainThread();
+    GTMouseDriver::dragAndDrop(pointA, pointGrpB);
+
+    // 8. Drag&drop group #1 to group #2
+    pointGrpA = GTTreeWidget::getItemCenter(os, grpA);
+    pointGrpB = GTTreeWidget::getItemCenter(os, grpB);
+    GTThread::waitForMainThread();
+    GTMouseDriver::dragAndDrop(pointGrpA, pointGrpB);
+
+    // Expected: group moved successfully, no crash
```

Comment:
Check the result state. Otherwise even if dragAndDrop fails with no crash the test will pass.

Reply:
done